kubernetes/linux/main.sh

```diff
@@ -1060,9 +1060,10 @@ if [ ! -f /etc/cron.d/ci-agent ]; then
       echo "*/5 * * * * root /usr/sbin/logrotate -s /var/lib/logrotate/ci-agent-status /etc/logrotate.d/ci-agent >/dev/null 2>&1" >/etc/cron.d/ci-agent
 fi
 
-setGlobalEnvVar AZMON_WINDOWS_FLUENT_BIT_DISABLED "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}"
+# setGlobalEnvVar AZMON_WINDOWS_FLUENT_BIT_DISABLED "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}"
+setGlobalEnvVar AZMON_WINDOWS_FLUENT_BIT_DISABLED "$( [ "${AZMON_RESOURCE_OPTIMIZATION_ENABLED}" == "true" ] && echo "false" || echo "true" )"
 if [ "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}" == "true" ] || [ -z "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}" ] || [ "${USING_AAD_MSI_AUTH}" != "true" ] || [ "${RS_GENEVA_LOGS_INTEGRATION}" == "true" ]; then
-      if [ -e "/etc/config/kube.conf" ]; then
```

Comment:
This check - [ "${RS_GENEVA_LOGS_INTEGRATION}" == "true" ]; needs to be removed for cosmic to get the metrics collected through Geneva. let me test this change quickly.

Reply:
branch [longw/win-ama-resource-optimization-flag-cosmic-geneva]